extensions/map-layers/src/ui/widget/MapLayerManager.tsx

```diff
@@ -169,6 +171,7 @@ function getMapLayerSettingsFromStyle(displayStyle: DisplayStyleState | undefine
         subLayers: populateSubLayers ? getSubLayerProps(layerSettings.subLayers) : undefined,
         showSubLayers: false,
         isOverlay: false,
+        mutualExclusiveSubLayer: (layerSettings.formatId === "WMTS"),
```

Comment:
I suppose its unlikely we'll support any more formats - but rather than having this testing for WMTS explicitly shouldn't mutualExclusiveSubLayer just be a property of the provider (which defaults to false and is overriden by WMTS).

Reply:
If Im not mistaken, at this point the layers settings come from the displayStyle, and the provider is not directly accessible.. so that means I will have to create a MapLayerImagerProvider object and keep in on the StyleMapLayerSettings (or create it on the fly everytime).  Is that what you meant?